package.json

```diff
@@ -110,6 +109,7 @@
     "sass-resources-loader": "^2.0.1",
     "source-map-loader": "^4.0.1",
     "standard-version": "^9.0.0",
+    "storybook": "^7.0.23",
```

Comment:
Suggestion(nit-pick): Since there are other requested changes would you mind bumping this to the latest storybook 7.1.0?